# Table of Contents: 


## Cell-free DNA General Background 
* Pan et al ["Brain Tumor Mutations Detected in Cerebral Spinal Fluid"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Pan%20et%20al.pdf)
* Schilling and Rehli ["Global, comparative analysis of tissue-specific 
promoter CpG methylation"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Schilling%20and%20Rehli.pdf)
* Jahr et al ["DNA Fragments in the Blood Plasma of Cancer 
Patients: Quantitations and Evidence for Their Origin from Apoptotic 
and Necrotic Cells"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Jahr%20et%20al%20.pdf)
* Van der Vaart and Pretorius ["Circulating DNAIts Origin and Fluctuation"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Van_der_Vaart_and_Pretorious.pdf)
* Stroun et al ["The Origin and Mechanism of Circulating DNA"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Stroun%20et%20al.pdf)
* Gravina et al ["The dark side of circulating nucleic acids" ](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Gravina%20et%20al.pdf)
* Sun et al ["Plasma DNA tissue mapping by genome-wide methylation 
sequencing for noninvasive prenatal, cancer, and 
transplantation assessments"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Sun_et_al.pdf)
* Wyatt et al ["Concordance of Circulating 
Tumor DNA and MatchedMetastatic Tissue 
Biopsy in Prostate Cancer"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Wyatt_et_al.pdf)
* Ma et al ["Cell-Free DNA Provides a Good Representation 
of the Tumor Genome Despite 
Its Biased Fragmentation Patterns"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Ma_et_al.pdf)
* Luck et al ["A Stochastic Model for the Formation of Spatial
Methylation Patterns"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Luck_preprint.pdf)
* Doherty and Couldrey ["Exploring genome wide bisulfite sequencing for DNA methylation analysis in livestock: a technical assessment"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/Doherty_and_Couldrey.pdf)
* Intro methylation review: ["DNA methylation: a form of 
epigenetic control of gene expression"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_background/intro_methylation_review.pdf)


## Papers Relating to Tissue Deconvolution
* Lehman-Werman et al  ["Identification of tissue-specific cell 
death using methylation patterns of circulating DNA"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/tissue_specific_cfDNA/Lehman-Werman_et_al.pdf)
* Synder et al ["Cell-free DNA Comprises an In Vivo Nucleosome
Footprint that Informs Its Tissues-Of-Origin"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/tissue_specific_cfDNA/Synder_et_al.pdf)
* Kessler et al ["CpG methylation differences between 
neurons and glia are highly 
conserved from mouse to human"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/tissue_specific_cfDNA/Kessler_et_al.pdf)
* Jensen et al ["Whole genome bisulfite sequencing of 
cell-free DNA and its cellular contributors 
uncovers placenta hypomethylated domains"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/tissue_specific_cfDNA/Jensen_et_al.pdf)
* Guo et al ["Identification of methylation haplotype 
blocks aids in deconvolution of heterogeneous 
tissue samples and tumor 
tissue-of-origin mapping from plasma DNA"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/tissue_specific_cfDNA/Guo_et_al.pdf)


## Algorithm development 
* Lee et al ["An integrative approach for efficient 
analysis of whole genome bisulfite sequencing data"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Lee_et_al.pdf)
* Kunde-Ramamoorty et al ["Comparison and quantitative 
verification of mapping algorithms 
for whole-genome bisulfite sequencing"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Kunde-Ramamoorthy_et_al.pdf)
* Xi and Li ["BSMAP: whole genome bisulfite 
sequence MAPping program"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Xi%20and%20Li.pdf)
* Guo et al "BS-Seeker2: a versatile aligning pipeline for bisulfite sequencing data" 
* Houseman et al ["Reference-free 
deconvolution of DNA methylation 
data and mediation by cell 
composition effects" ](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Houseman_et_al.pdf)
* Rahmani et al ["Sparse PCA Corrects for 
Cell-Type Heterogeneity in Epigenome-Wide 
Association Studies"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Rahmani_et_al.pdf)
* Gond and Szustakowski ["DeconRNASeq: a statistical framework for 
deconvolution of heterogeneous 
tissue samples based on mRNA-Seq data"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Gong%20and%20Szustakowski.pdf)
* Krueger and Andrews ["Bismark: a 
flexible aligner and methylation 
caller for Bisulfite-Seq applications"](https://github.com/christacaggiano/cell-free-dna-reading-list/blob/master/cfDNA_tissue_of_origin_deconvolution/Krueger%20and%20Andrews.pdf)
